pacstall

```diff
@@ -196,7 +196,9 @@ while [[ "$1" =~ ^- && ! "$1" == "--" ]]; do
         fancy_message error "Run as root"
         exit 1
       fi
-      PACKAGE=$2
+      PACKAGE="$@"
+      PACKAGE=$(echo $PACKAGE | cut -dI -f2- | sed 's/ //')
+      for PACKAGE in $PACKAGE; do
```

Comment:
No `done` after `do`

Reply:
?

Reply:
for i in {1,2,3}; do echo $i ; done

Reply:
There is no `done` statement to close this `do` block. I mean you opened the `for` loop but you didn't end it.

Reply:
At the very end, I added `done` before the exit

Reply:
Line number?

Reply:
237

Reply:
237 is under `-Il` case I am talking about the `for` loop of `-I` case (line number 201)